### ISSUE TITLE
NEWS incorrectly states when Python 2.6 was dropped

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8651]
|[<img alt="glasnt's avatar" src="https://avatars.githubusercontent.com/u/813732?s=50" width="50" height="50">](https://github.com/glasnt)| @glasnt reported|
|-|-|
|Trac ID|trac#8651|
|Type|task|
|Created|2016-07-22 08:57:03Z|

Python 2.6 was dropped in 15.5, but the NEWS file says that 15.5 was the last version to support Python 2.6

[#8017](https://github.com/twisted/twisted/issues/8017)

https://github.com/twisted/twisted/blob/1d7df3fdad132ee3d091c01e02cd6c196de112fd/NEWS#L570

<details><summary>Searchable metadata</summary>

```
trac-id__8651 8651
type__task task
reporter__glasnt glasnt
priority__low low
milestone__None None
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469177823593344 1469177823593344
changetime__1469181085161947 1469181085161947
version__None None
owner__Lukasa Lukasa

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="glasnt's avatar" src="https://avatars.githubusercontent.com/u/813732?s=50" width="50" height="50">](https://github.com/glasnt)<a name="note_1"></a>|@glasnt commented|
|-|-|

https://github.com/twisted/twisted/pull/390

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_2"></a>|@Lukasa commented|
|-|-|

Sorry Katie, but where do you draw the conclusion that 2.6 support was dropped in Twisted 15.4? GitHub identifies the first tagged release containing commit e748819bfd0faf8fb8f7afafa3fc12bb7b7a1eb4 (the merge commit for ticket [#8017](https://github.com/twisted/twisted/issues/8017)) as being 15.5, as you can see here: https://github.com/twisted/twisted/commit/e748819bfd0faf8fb8f7afafa3fc12bb7b7a1eb4.

**Comment by twisted-trac** [comment]
|[<img alt="glasnt's avatar" src="https://avatars.githubusercontent.com/u/813732?s=50" width="50" height="50">](https://github.com/glasnt)|@glasnt removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_4"></a>|@Lukasa set owner to @Lukasa|
|-|-|

Katie has kindly explained to me my misunderstanding: the commit I linked to is the one that *drops* Python 2.6 support. That means that, by definition, Python 2.6 isn't supported in 15.5. With that in mind, then, I'm happy with this. =) I looked upon this patch and I saw that it was good.

I'll merge this.

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_5"></a>|@Lukasa set owner to @glasnt|
|-|-|

Ah, Travis correctly points out that we're missing a topfile for this!

Katie, each change needs to be accompanied by a 'topfile', which is a file that can be aggregated together to create our release notes. The full discussion of topfiles is [here](https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles), but the TL;DR is: can you create a file in the twisted/topfiles directory called "8651.doc", and add one line describing this change? Something like "Updated release notes to reflect that 15.4 is the last version that supported Python 2.6, not 15.5."

When that's done, please submit this ticket for review again. =)

**Comment by twisted-trac** [comment]
|[<img alt="glasnt's avatar" src="https://avatars.githubusercontent.com/u/813732?s=50" width="50" height="50">](https://github.com/glasnt)|@glasnt removed owner|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="Lukasa's avatar" src="https://avatars.githubusercontent.com/u/1382556?s=50" width="50" height="50">](https://github.com/Lukasa)<a name="note_7"></a>|@Lukasa set owner to @Lukasa|
|-|-|

I pronounce both myself and Travis happy with this patch. I'll merge.

Thank you so much Katie!

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_8"></a>|GitHub set status to `closed`|
|-|-|

In changeset da03223ef5309a7184b3e590ca773154ebd3a77c
```
#!CommitTicketReference repository="" revision="da03223ef5309a7184b3e590ca773154ebd3a77c"
Merge pull request #390: Move notice about Python 2.6 from 15.5 to 15.4

Author: glasnt
Reviewer: Lukasa
Fixes: #8651
```
